colossalai/booster/plugin/hybrid_parallel_plugin.py

```diff
@@ -38,13 +38,13 @@ def _convert_floating_point(x, dtype: torch.dtype = torch.float16):
 class HybridParallelModule(ModelWrapper):
 
     def __init__(self, module: Module, precision: str, shard_config: ShardConfig, dp_group: ProcessGroup, use_ddp: bool,
-                 ddp_config: dict) -> None:
```

Comment:
It's best not to expose 'Policy' in the plugin; it's just a component of Shardformer.

Reply:
Hi,

Thanks for your review!

I have two candidate approaches currently.
1. Remove the typing hint
2. Modify the typing hint from Policy to Object

Which one do you think better?

Reply:
Hi,

I modify the code with the second approach (modify from `Policy` to `object`)